inst/Description.qml

```diff
@@ -16,13 +16,13 @@ Description
 	Analysis
 	{
 		title:	qsTr("Principal Component Analysis")
-		func:	"PrincipalComponentAnalysis"
+		func:	"principalComponentAnalysis"
```

Comment:
so, QML files should be capitalised (QML uses PascalCase). It's ok to rename the R function to adhere to camelCase style we follow in R, but the QML file should stay capitalized. You will just need to add the `qml` file in here like this (see https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/jasp-adding-module.md#analysis):

```suggestion
		func:	"principalComponentAnalysis"
		qml:	"PrincipalComponentAnalysis.qml"
```

Reply:
the same goes for `exploratoryFactorAnalysis` and `ExploratoryFactorAnalysis.qml`